x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlQueryRequest.java

```diff
@@ -232,12 +232,13 @@ public Map<String, Map<String, Column>> tables() {
         return tables;
     }
 
-    public boolean allowPartialResults() {
+    public Boolean allowPartialResults() {
         return allowPartialResults;
     }
 
-    public void allowPartialResults(boolean allowPartialResults) {
+    public EsqlQueryRequest allowPartialResults(boolean allowPartialResults) {
```

Comment:
Doesn't look like we're using this return value anywhere?

Reply:
Yes, I tried to make it a fluent pattern (like other request builders).